app/lib/widget/Common.js

```
@@ -582,34 +587,36 @@ function run(task, message, indicator) {
+        }
+        document.body.style.fontSize = size;
+    }
 
-                    var family = Config.get(Config.UI_CUSTOM_FONT_FAMILY);
-                    if (family) document.body.style.fontFamily = family;
+    var family = Config.get(Config.UI_CUSTOM_FONT_FAMILY);
+    if (family) document.body.style.fontFamily = family;
 
-                    var size = Config.get(Config.UI_CUSTOM_FONT_SIZE);
-                    if (size) document.body.style.fontSize = size;
+    var size = Config.get(Config.UI_CUSTOM_FONT_SIZE);
```

Biome (2.4.6)

[error] Shouldn't redeclare 'size'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)